maintainers/maintainer-list.nix

```diff
@@ -5312,12 +5297,33 @@
     githubId = 39689;
     name = "Hugo Tavares Reis";
   };
+  hufman = {
+    email = "hufman@gmail.com";
+    github = "hufman";
+    githubId = 1592375;
+    name = "Walter Huf";
+  };
```

Comment:
Apologies, I thought this wasn't sorted. Please don't make changes to locations of other attributes as it creates big diffs that are hard to read. You can revert changes to your previous iteration.

Reply:
In this case it only moves 2 other users to be in the proper order, is that still too big of a diff?

Reply:
You shouldn't _refactor_ this file, but rather just try to "sort" your entry upon adding it. Given the fact that a lot of people could have open PRs that modify this file, keeping your changes small will make it easier to resolve and avoid merge conflicts. It's the same reason why `all-packages.nix` is in a messy state: I bet no one likes that, but there's not much that could be done that will not break the _GitHub flow_.

Reply:
Very good!